user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Configuring_external_authentication_via_an_identity_provider_using_SAML.md

```diff
@@ -101,6 +101,36 @@ To create a DataMiner metadata file (also referred to as *Service Provider Metad
 > The ``WantAssertionsSigned`` flag is supported as from DataMiner version 10.2.1/10.2.0. If you are using an older version, then set this to false.
 > SAML responses without signatures can be freely edited to tamper with permissions on the application, leading to severe vulnerabilities. We **highly recommend** setting ``WantAssertionsSigned`` to *true* to mitigate this.
 
+## Additional configuration for systems connected to dataminer.services
+
+When your DataMiner System is connected to dataminer.services, additional configuration is required for both the *spMetadata.xml* file and the identity provider.
+
+1. Add the following URLs:
+
+   - `https://<dms-dns-name>-<organization-name>/API/`
+   - `https://<dms-dns-name>-<organization-name>/account-linking`
+   - `https://<dms-dns-name>-<organization-name>/account-linking/`
```

Comment:
```suggestion
   - `https://<dms-dns-name>-<organization-name>/account-linking/`
   
     > [!NOTE]
     > Note the trailing "/".
```
Perhaps we could add this note to make it clear to readers that there is a significant difference between URL 2 and 3?